adoc/iii-Business/SA.adoc

```diff
@@ -63,6 +63,14 @@ SUSE Engineering continue to develop new scenarios based on the demands of custo
 
 The overall deployment using SUSE SAP Automation looks as follows:
 
+////
+- The HA NFS Service with DRBD is used in Azure and GCP environments. AWS environment uses EFS file shares
+- The iSCSI server provides SBD disks is used only in the Azure environment. GCP and AWS environments use the cloud fencing agents (`fence_gce` and `fence_aws`)
+
+I would suggest one of the following:
+  1. Using separate diagrams for GCP and AWS. @Peter If you can share the original LibreOffice Impress slide, that would be great.
+  2. Mentioning that the HA NFS Service with DRBD and the iSCSI server are optional components depend on the Cloud Service Providers.
+////
```

Comment:
The overall architecture does not differ. And yes, i have a slide more neutral. Its in the presentations - and the deck is shared since August 2020.

Reply:
Thanks, Peter. I will add some update this part to match the GCP infrastructure.

Reply:
so have a look at the presentation, there is a second picture which show "services" instead of the HA-NFS-DRBD and fencing. We need to simply change the picture